src/www/ui/api/Middlewares/RestAuthMiddleware.php

```diff
@@ -70,8 +70,21 @@ public function __invoke($request, $response, $next)
         $tokenValid = new Info(403, "Do not have required scope.", InfoType::ERROR);
       }
       if ($tokenValid === true) {
-        $authHelper->updateUserSession($userId, $tokenScope);
-        $response = $next($request, $response);
+        $groupName = "";
+        $groupName = strval($request->getHeaderLine('groupName'));
+        if (!empty($groupName)) { // if request contains groupName
+          $userHasGroupAccess = $authHelper-> userHasGroupAccess($userId, $groupName);
```

Comment:
Just a little extra space.
```suggestion
          $userHasGroupAccess = $authHelper->userHasGroupAccess($userId, $groupName);
```